src/components/community/Boards/BoardSearch.tsx

```diff
@@ -7,29 +7,32 @@ import SectionTitle from '@/components/community/SectionTitle'
 import SearchBox from '@/components/timetable/SearchBox'
 import NoticeModal from '@/components/ui/modal/NoticeModal'
 import { COMMUNITY_SEARCH_MESSAGES } from '@/lib/messages/community'
+import { useQueryParams } from '@/util/hooks/useQueryParams'
 import useIntersect from '@/util/useIntersect'
 import { useModal } from '@/util/useModal'
-import { useSearch } from '@/util/useSearch'
 
 const boardConfig: { [key: string]: number } = {
   community: 1,
   question: 2,
   information: 3,
 }
+type SearchParams = {
+  keyword: string
```

Comment:
```suggestion
  keyword?: string
```
얘를 이렇게 수정해서

```ts
setQueryParams(keyword as SearchParams)
```
여기에서 as 안 쓰게 하는게 어떨까요 (실제로 keyword값은 undefined일 수 있으니까)